tests/rubinrepoman_test.py

```diff
@@ -4,17 +4,66 @@
 registry, getting the list of available tags, and filtering
 that down into a list of images to pull.
 """
-
 from __future__ import annotations
 
 from typing import List
 
 from cachemachine.rubinrepoman import RubinRepoMan
-from cachemachine.types import CachedDockerImage
+from cachemachine.types import CachedDockerImage, DockerImage
 
+from .conftest import mock_registry
 from .docker_mock import DockerMock
 
-HOST = "registry.hub.docker.com"
+repo = "registry.hub.docker.com/lsstsqre/sciplat-lab"
+expected_names = {
+    "prepuller_pulled_recommended": "prepuller_pulled_recommended",
+    "recommended": "Recommended (Release r21.0.0)",
+    "r22_0_0_rc1": "Release Candidate r22.0.0-rc1",
+    "r21_0_0": "Release r21.0.0",
+    "w_2021_22": "Weekly 2021_22",
+    "w_2021_21_c0021.003": "Weekly 2021_21 (SAL Cycle 0021, Build 003)",
+    #    "w_2021_22_c0021.003_20210514": "Weekly 2021_21_c0021.003_20210514",
```

Comment:
is this commented out and should be removed?

Reply:
Rather, we should test those too.  Done.